### PR TITLE
data:lilmubby

### PR DESCRIPTION
**Pull request by lilmubby**
## Fixes Issue
added my data linkfree

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x ] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3588"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

File: data/lilmubby.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/mubbyszn",
+      "icon": "FaInstagram"
+    },{
+      "name": "Let's Connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/mubarak-yahaya-3548271a5/",
+      "icon": "FaLinkedIn"
```
Comment (Pradumnasaraf) [inline data/lilmubby.json:20]:
```suggestion
      "icon": "FaLinkedin"
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

`i` in Linkedin should be lowercase
Screenshot 2023-01-18 at 1 19 50 AM

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**